tests/sync/util.js

```diff
@@ -147,6 +147,7 @@ const syncFromDisk = require('../../sync/syncFromDisk');
  * @property {string[]=} serverFilesCourse
  * @property {number=} timeout
  * @property {boolean=} enableNetworking
+ * @property {Object=} environment
```

Comment:
Nit, this is more precise:
```suggestion
 * @property {Record<string, string | null>=} environment
```